tests/performance/README.md

```diff
@@ -2,7 +2,8 @@
 
 usage:
 ```
-Usage: ./build/tests/test_performance <dataset_file_path> <process> <index_name> <build_param> <search_param>
+build: ./build/tests/test_performance <dataset_file_path> "build" <index_name> <build_param> <search_param>
+search: ./build/tests/test_performance <dataset_file_path> "search:<top-k>" <index_name> <build_param> <search_param>
```

Comment:
Q: Why in search mode need provide `build_param` ？

Reply:
Currently, the index does not support self-description, so the constructor of the index needs to be used before the serialization phase.